pkg/client/cluster.go

```diff
@@ -583,7 +583,7 @@ func ClusterDelete(ctx context.Context, runtime k3drt.Runtime, cluster *k3d.Clus
 				if net == cluster.Network.Name {
 					continue
 				}
-				if net == "bridge" || net == "host" {
+				if net == k3d.DefaultNetwork || net == "host" {
```

Comment:
```suggestion
				if net == k3d.DefaultRuntimeNetwork || net == "host" {
```
Making clear, that it's not the default k3d network, but the default network as specified by the (docker) runtime.